### DISCUSSION TITLE
A Pythia mailing list?

### DISCUSSION THREAD

**Discussion by DamienIrving** [ProjectPythia/projectpythia.github.io#208] [Ideas]
I was wondering if the Pythia team has thought about hosting a mailing list for community announcements etc?

At the moment I'm moderating the [PyAOS mailing list](http://lists.johnny-lin.com/listinfo.cgi/pyaos-johnny-lin.com), which has about 600 subscribers. People typically post conference announcements, new releases of relevant Python software packages, etc. The mailing list is still hosted at johnny-lin.com and when Johnny handed it over to me a couple of years ago I said I'd try and find an alternative place to host it. 

At the same time, I've had a few people suggest a Discourse-style site instead of a mailing list so people can post questions basic PyAOS questions and get answers. I briefly tried out [peerboard](https://pyaos.peerboard.com/) as a free alternative to Discourse but I'm honestly not sure whether a place where people can ask general questions is a good idea or not (Stack Overflow and package specific help channels might be more appropriate). I'm also not sure that free platform that hasn't been around very long is a good bet long term.

Given that there's a huge overlap between the PyAOS and Pythia audience (and given that I'm looking to host that PyAOS mailing list elsewhere anyway), I'd thought I'd raise the idea of transitioning the PyAOS mailing list over to being a Pythia mailing list. (Or if not, perhaps pursuing an alternative discourse-style platform like Pangeo has?)

**Comment by brian-rose** [comment]
Hi @DamienIrving! There is a mailing list at projectpythia@ucar.edu. I think it's currently only going out to a small group of core Pythia contributors. And its existence may be a well-kept secret, since I can't actually find any mention of this list on the [Pythia portal](https://projectpythia.org). We have been using it mostly to announce meetings.

What do folks think about a broader-scoped Pythia community mailing list?

**Comment by clyne** [comment]
The projectpythia@ucar.edu mailing list is found via the 'Email us' link in the footer of the [Pythia portal](https://projectpythia.org/). Currently, I believe that the only subscribers are the pythia maintainers (and I'm not even sure that it is possible to open this up to the outside world because of the @ucar.edu address). @jukent knows more about this. We've been promoting GitHub Discussion forum that you've posted to as an alternative to a broadly subscribed mailing list for reaching the Pythia community. But currently, as you can see, there is not a lot of activity here. 

@DamienIrving what might "transitioning the PyAOS mailing list over to being a Pythia mailing list" look like? Do you think the Pythia GitHub Discussion forum is a possibility, or are you thinking of a traditional mailing list? I'm intrigued by the idea of bootstrapping discussion on a Pythia branded resource with an established community of Earth Sciences Python users.

**Reply by jukent** [reply]
Yes the projectpythia@ucar.edu mailing list is being used for contributors only. We could make a separate list for customers that (despite a "@ucar.edu" domain) could be open to anyone to subscribe to.

**Reply by DamienIrving** [reply]
The PyAOS mailing list uses [GNU Mailman](https://list.org/) under the hood, so one option would be to simply [move that list to the UCAR server](https://wiki.list.org/DOC/How%20do%20I%20move%20a%20list%20to%20a%20different%20server-Mailman%20installation.). The mailing address would simply change from pyaos@lists.johnny-lin.com to pyaos@lists.ucar.edu. We could also change the name to pythia@lists.ucar.edu if we wanted. The advantage of a straight transfer is that none of the 600 current subscribers would be lost. If you wanted to use a different mailing list client we could email the current PyAOS list and encourage them to sign up to the new list, but I'm pretty sure we'd lose a large fraction of the existing 600 subscribers in the process.

I really like the idea of using GitHub Discussions instead of a traditional mailing list, however as far as I understand when you watch a repo on GitHub you can't separate out Discussions email notifications from all the other notifications (of new issues, PRs, etc) - see [this thread](https://github.com/github/feedback/discussions/59#discussioncomment-352736) about this problem. Until that problem is fixed it's a pretty big barrier (IMHO) to using GitHub Discussions as a replacement for a traditional mailing list.
